design/cli-install-changes.md

```diff
@@ -0,0 +1,286 @@
+      --wait                                       wait for Velero deployment to be ready. Optional.
+```
+
+Flags moved to...
+
+...`backup-location create`
+```
+      --backup-location-config mapStringString     configuration to use for the backup storage location. Format is key1=value1,key2=value2
+      --bucket string                              name of the object storage bucket where backups should be stored
+      --prefix string                              prefix under which all Velero data should be stored within the bucket. Optional.
+      --dry-run                                    generate resources, but don't send them to the cluster. Use with -o. Optional.
+```
+
+...`snapshot-location create`
+```
+      --snapshot-location-config mapStringString   configuration to use for the volume snapshot location. Format is key1=value1,key2=value2
+```
+
+...both `backup-location create` and `snapshot-location create`
+```
+      --provider string                            provider name for backup and volume storage 
+      --no-secret                                  flag indicating if a secret should be created. Must be used as confirmation if --secret-file is not provided. Optional.
+      --secret-file string                         file containing credentials for backup and volume provider. If not specified, --no-secret must be used for confirmation. Optional.
+      --dry-run                                    generate resources, but don't send them to the cluster. Use with -o. Optional.
+```
+
+...`restic`
+```
+      --default-restic-prune-frequency duration    how often 'restic prune' is run for restic repositories by default. Optional.
+      --restic-pod-cpu-limit string                CPU limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --restic-pod-cpu-request string              CPU request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --restic-pod-mem-limit string                memory limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --restic-pod-mem-request string              memory request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --use-restic                                 create restic deployment. Optional.
+```
+
+...`plugin`
+ ```
+      --plugins stringArray                        Plugin container images to install into the Velero Deployment
+      --sa-annotations mapStringString             annotations to add to the Velero ServiceAccount. Add iam.gke.io/gcp-service-account=[GSA_NAME]@[PROJECT_NAME].iam.gserviceaccount.com for workload identity. Optional. Format is key1=value1,key2=value2
+```
+
+##### Velero Server
+`velero server (DEPRECATED)` 
```

Comment:
we can't deprecate this command, this is the "internal" command that the velero server pod runs :) So `velero init` would be the command that the user runs, that would create a Deployment whose pod runs the `velero server` command, with all the appropriate flags.

Reply:
As Steve said, this is the piece of Velero that runs the control loops that process CRDs and actually does the backups/restores/scheduling.

Reply:
Ah yes, I should've made my intention more clear here. I meant to rename this to `velero init`,  so deprecation in name only, since we'd keep the same behavior (as `velero init`). Just to double check:

>So velero init would be the command that the user runs

Not clear if this means it should be `velero init server` or if we can s/server/init.

Reply:
I think just `velero init` would be fine, though I suppose you could also go with `velero init server` & `velero init restic`

Reply:
Just circling back on this, I think we still need the two separate commands:

`velero config server` - a user runs this, and it generates/applies the YAML
`velero server` - this is the command run by the Velero pod, that starts the velero server (see https://github.com/vmware-tanzu/velero/blob/master/pkg/install/deployment.go#L145-L150)

Reply:
Yes, this clarification is useful. I was not considering the `velero server` cmd in the deployment itself.